src/modules/games/games.service.ts

```diff
@@ -194,7 +195,17 @@ export class GamesService {
       game.mode = mode;
     }
 
-    return this.gamesRepository.save({ ...game, ...updateGameDto });
+    const fighters: Fighter[] = [];
+
+    for (const fighter of updateGameDto.fighters) {
+      fighters.push(await this.fighterService.create(fighter));
+    }
+
+    if (game.fighters.length > 0) {
+      await this.fighterService.remove(game.fighters);
+    }
+
```

Comment:
```suggestion
game.fighters = fighters
return this.gamesRepository.save({ ...game, ...updateGameDto, fighters });
```

Ca suffit pas ça pour mettre à jour les fighters d'une game ? Et la création des données dans la table fighters ?

Reply:
en faite ce sont de nouveau fighters donc ca va bien remplacer mais les anciens vont être orphelins dans la bdd (déjà test) donc faut remove au préalable

Reply:
Mais le "cascade" ne gère pas ça ?

Reply:
si tu mets cascade="delete" ca fonctionne quand tu supprimes l'enfant ca supprime le parent dans notre cas on veut supprime le parent donc ca trigger pas :/ de ce que j'en ai conclu